library/ssl_tls13_generic.c

```diff
@@ -494,56 +494,53 @@ void mbedtls_ssl_set_outbound_transform( mbedtls_ssl_context *ssl,
 }
 #endif /* !MBEDTLS_SSL_USE_MPS */
 
-    /*
-     * The mbedtls_ssl_create_verify_structure() creates the verify structure.
-     * As input, it requires the transcript hash.
-     *
-     * The structure is computed per TLS 1.3 specification as:
-     *   - 64 bytes of octet 32,
-     *   - 33 bytes for the context string
-     *        (which is either "TLS 1.3, client CertificateVerify"
-     *         or "TLS 1.3, server CertificateVerify"),
-     *   - 1 byte for the octet 0x0, which servers as a separator,
-     *   - 32 or 48 bytes for the Transcript-Hash(Handshake Context, Certificate)
-     *     (depending on the size of the transcript_hash)
-     *
-     * This results in a total size of
-     * - 130 bytes for a SHA256-based transcript hash, or
-     *   (64 + 33 + 1 + 32 bytes)
-     * - 146 bytes for a SHA384-based transcript hash.
-     *   (64 + 33 + 1 + 48 bytes)
-     *
-     * The caller has to ensure that the buffer has this size.
-     */
-static void mbedtls_ssl_create_verify_structure(
-                                        unsigned char *transcript_hash,
+/*
+ * The ssl_create_verify_structure() creates the verify structure.
+ * As input, it requires the transcript hash.
+ *
+ * The structure is computed per TLS 1.3 specification as:
+ *   - 64 bytes of octet 32,
+ *   - 33 bytes for the context string
+ *        (which is either "TLS 1.3, client CertificateVerify"
+ *         or "TLS 1.3, server CertificateVerify"),
+ *   - 1 byte for the octet 0x0, which servers as a separator,
+ *   - 32 or 48 bytes for the Transcript-Hash(Handshake Context, Certificate)
+ *     (depending on the size of the transcript_hash)
+ *
+ * This results in a total size of
+ * - 130 bytes for a SHA256-based transcript hash, or
+ *   (64 + 33 + 1 + 32 bytes)
+ * - 146 bytes for a SHA384-based transcript hash.
+ *   (64 + 33 + 1 + 48 bytes)
+ *
+ * The caller has to ensure that the buffer has this size.
+ */
+static void ssl_create_verify_structure( unsigned char *transcript_hash,
                                         size_t transcript_hash_len,
                                         unsigned char *verify_buffer,
                                         size_t *verify_buffer_len,
                                         int from )
 {
-    /* The length of context_string_[client|server] is
-     * sizeof( "TLS 1.3, xxxxxx CertificateVerify" ) - 1, i.e. 33 bytes.
-     */
-    const unsigned int content_string_len = sizeof( "TLS 1.3, xxxxxx CertificateVerify" ) - 1;
-    const unsigned char context_string_client[] = "TLS 1.3, client CertificateVerify";
-    const unsigned char context_string_server[] = "TLS 1.3, server CertificateVerify";
+    size_t i = 64;
```

Comment:
Give this a descriptive name. E.g.:
```c
uint8_t const verify_padding_val = 64;
size_t const verify_padding_len = 32;
size_t i = 0;
memset( verify_buffer, verify_padding_val, verify_padding_len );
i += verify_padding_len;
```

Reply:
@lhuang04 Sorry, I might not have been clear: I meant to give a name to the `64`, not necessarily change the name of `i`.